packages/cli/src/commands/ui/create/react.ts

```diff
@@ -123,11 +123,25 @@ export default class React extends Command {
     return new Config(this.config.configDir, this.error);
   }
 
-  private async getUserInfo() {
+  private async platformUserCredentials() {
+    const args = this.args;
     const authenticatedClient = new AuthenticatedClient();
     const platformClient = await authenticatedClient.getClient();
     await platformClient.initialize();
 
-    return await platformClient.user.get();
+    const userInfo = await platformClient.user.get();
+    const apiKey = await authenticatedClient.createImpersonateApiKey(args.name);
+
+    return {userInfo, apiKey};
+  }
```

Comment:
This method could be a function if it took an ApiKeyName param.

If so, you could refactor it and avoid repetition in several places.